README.md

```diff
@@ -22,26 +30,78 @@ browzine.script.src = "https://s3.amazonaws.com/browzine-adapters/summon/browzin
 document.head.appendChild(browzine.script);
 ```
 
+_BrowZine 360 Core Adapter Script_
+
+```
+var browzine = {
+  api: "https://api.thirdiron.com/public/v1/libraries/XXX",
+  apiKey: "ENTER API KEY",
+};
+
+browzine.script = document.createElement("script");
+browzine.script.src = "https://s3.amazonaws.com/browzine-adapters/360-core/browzine-360-core-adapter.js";
+document.head.appendChild(browzine.script);
+```
+
+_Loading Discovery Service Adapters_
```

Comment:
I'm not quite following how this is different than the above two adapter scripts... who is the intended audience for this code snippet?  Seems like the 360 Core and Summon code snippets above cover both of these cases.